docs/type-definitions/unions.md

```diff
@@ -16,13 +16,13 @@ $searchResultType = new UnionType([
         MyTypes::story(),
         MyTypes::user()
     ],
-    'resolveType' => function($value) {
-        if ($value->type === 'story') {
-            return MyTypes::story();            
-        } else {
-            return MyTypes::user();
+    'resolveType' => function (object $value): ObjectType {
```

Comment:
It is assumed to be an `object` here, since the subsequent code accesses the property `->type` on it.

Reply:
I know but static analysers scream because of covariance violation so not sure if doing it is the best practice to be put into docs. `resolveTypeFn` should be able to always accept `mixed`. I'd rather handle the type within function body.

Reply:
Changed.